beacon-chain/core/epoch/epoch_operations.go

```diff
@@ -87,31 +79,17 @@ func BoundaryAttestations(
 // (state.slot - 2 * EPOCH_LENGTH...state.slot - EPOCH_LENGTH).
 //
 // Spec pseudocode definition:
-//   return [a for a in state.latest_attestations
-//   if state.slot - 2 * EPOCH_LENGTH <= a.slot < state.slot - EPOCH_LENGTH]
+//   return [a for a in state.latest_attestations if
+//   	previous_epoch == slot_to_epoch(a.data.slot)].
 func PrevAttestations(state *pb.BeaconState) []*pb.PendingAttestationRecord {
-	epochLength := params.BeaconConfig().EpochLength
 	var prevEpochAttestations []*pb.PendingAttestationRecord
-	var earliestSlot uint64
-	var lastSlot uint64
+	var prevEpoch uint64
 
+	if helpers.CurrentEpoch(state) != 0 {
```

Comment:
Can you put this piece in the helper package as `prevEpoch` ? We can then reuse somewhere else if needed

Reply:
Great suggestion! Done! : )